src/server/schema/resolvers/mutations/createLife.ts

```diff
@@ -0,0 +1,29 @@
+import { ObjectId } from 'mongodb';
+import { getDatabaseContext, Life } from '../../../database';
+import { GraphQLMutationResolvers } from '../definitions';
+
+const mutation: GraphQLMutationResolvers['createLife'] = async (
+    root,
+    { firstName, lastName, birthday, description, hobbies, title }
+) => {
+    const { collections } = await getDatabaseContext();
+
+    const document: Life = {
+        _id: new ObjectId(),
+        firstName,
+        lastName,
+        hobbies,
+        birthday,
+        description,
+        title,
+    };
+
+    await collections.lives.insertOne(document);
+
+    return {
+        ...document,
+        fullName: `${firstName} ${lastName}`,
```

Comment:
it's not for your mutation to do such thing but you should implement the resolver for the type `Life` so you can implement a resolver for `Life.fullName`.